### DISCUSSION TITLE
Autocomplete without LSP?

### DISCUSSION THREAD

**Discussion by brandonchinn178** [helix-editor/helix#5264] [Ideas] [answered]
I generally don't use language servers, and Sublime Text still provides autocomplete for words in the same file (ignoring strings and comments), which is sufficient for me. Any thoughts on adding a built-in autocomplete?

**Comment by the-mikedavis** [comment, marked as answer]
This isn't implemented but there's an issue for it https://github.com/helix-editor/helix/issues/1015 and a PR https://github.com/helix-editor/helix/pull/3328

**Reply by brandonchinn178** [reply]
Silly me, only searching for `autocomplet.*` 😄
